### PR TITLE
Language will now set for site on load.

### PR DESCRIPTION
**Pull request by ocampd1**

### CODE REVIEW

File: chucks-pick3-web/angular-app/src/app/services/browser-locale.service.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+@Injectable({
+  providedIn: 'root'
+})
+export class BrowserLocaleService {
+  constructor(public translate: TranslateService) {
+  }
+
+  public findLanguage() {
+    let passedLanguage;
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/services/browser-locale.service.ts]: rename to `browserLanguage`.

File: chucks-pick3-web/angular-app/src/app/services/browser-locale.service.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    const recievedLanguage = navigator.language;
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/services/browser-locale.service.ts]: Don't need to create a variable here.  Just use `navigator.language' directly in if statement.

File: chucks-pick3-web/angular-app/src/app/shared/components/app.component/app.component.ts [hunk 3]
```diff
@@ -9,18 +10,27 @@ import {TranslateService} from '@ngx-translate/core';
...
     translate.setDefaultLang('en');
+    this.setLanguage();
   }
 
-  title = 'angular-app';
-  public showSplashPage = true;
 
-  switchLanguage(es: string) {
-    this.translate.use(es);
+  public setLanguage() {
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/app/shared/components/app.component/app.component.ts]: Renamed method to `useBrowserLanguage()`.

File: chucks-pick3-web/angular-app/src/assets/i18n/en.json [hunk 4]
```diff
@@ -158,5 +158,8 @@
   "detail-edit": "Edit",
   "detail-delete": "Delete",
   "detail-update": "Update",
-  "detail-cancel": "Cancel"
+  "detail-cancel": "Cancel",
+  "//n": "Splash Page------------------------------------------",
```
Comment (pouncilt) [inline chucks-pick3-web/angular-app/src/assets/i18n/en.json]: Remove this line.

File: chucks-pick3-web/angular-app/src/app/services/browser-locale.service.ts [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+import {TranslateService} from '@ngx-translate/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class BrowserLocaleService {
+  constructor(public translate: TranslateService) {
+  }
+
+  public findLanguage() {
```
[the remark this answers is not in the document]
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/services/browser-locale.service.ts]: Renamed to getLanguage.

File: chucks-pick3-web/angular-app/src/app/services/browser-locale.service.ts (hunk 1 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/services/browser-locale.service.ts]: Renamed to browserLanguage.

File: chucks-pick3-web/angular-app/src/app/services/browser-locale.service.ts (hunk 2 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/services/browser-locale.service.ts]: Removed constant and added directly to page.

File: chucks-pick3-web/angular-app/src/app/shared/components/app.component/app.component.ts (hunk 3 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/app/shared/components/app.component/app.component.ts]: Renamed method to useBrowserLanguage().

File: chucks-pick3-web/angular-app/src/assets/i18n/en.json [hunk 6]
```diff
@@ -158,5 +158,8 @@
   "detail-edit": "Edit",
```
[the remark this answers is not in the document]
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/assets/i18n/en.json]: Renamed keys

File: chucks-pick3-web/angular-app/src/assets/i18n/en.json (hunk 4 above)
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/assets/i18n/en.json]: Removed lines.

File: chucks-pick3-web/angular-app/src/assets/i18n/es.json [hunk 7]
```diff
@@ -158,6 +158,8 @@
   "detail-edit": "Editar",
   "detail-delete": "Eliminar",
   "detail-update": "Actualizar",
-  "detail-cancel": "Cancelar"
+  "detail-cancel": "Cancelar",
+  "//n": "Splash Page------------------------------------------",
```
[the remark this answers is not in the document]
Comment (ocampd1) [reply chucks-pick3-web/angular-app/src/assets/i18n/es.json]: Removed lines

**Comment by ocampd1** [comment]
Completed changes to branch.

**Pull request merged**